.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -62,6 +62,26 @@ body:
       description: "Add your log file or related error message. Again, please remove sensitive information."
     validations:
       required: false
+  - type: dropdown
+    attributes:
+      label: "Region"
+      description: "Choose the datacenter region where you encountered this issue."
+      multiple: false
```

Comment:
[pp] I would make multiple to true just in case it indeed happens in multiple regions